docs/rest.rst

```diff
@@ -128,6 +128,18 @@ Sending SMS'es
 Also known as :term:`MT SMS`, short for Mobile Terminated SMS, is when you want to
 deliver a SMS to a users mobile device.
 
+Message Filtering
+^^^^^^^^^^^^^^^^^
+
+Some messages contain links that due to phising attacks and generally unwanted spam cannot be accepted into the GatewayAPI, each account has a whitelist of links that are allowed, unique to that account, and approved by our staff. Any links found in the messages are checked against the whitelist, using the following method:
```

Comment:
Seems like the previous text was written with manual word wrapping, should I conform to that?